csm/core/agent/api.py

```diff
@@ -53,7 +53,7 @@
 from csm.core.controllers.view import CsmView, CsmResponse, CsmAuth
 from csm.core.controllers import CsmRoutes
 from cortx.utils.data.access import Query
-from cortx.utils.data.db.db_provider import (DataBaseProvider, GeneralConfig)
+from cortx.utils.data.db.db_provider import DataAccessError
```

Comment:
Changes required for this file has already been handled with this PR - https://github.com/Seagate/cortx-manager/pull/629.
No changes required in this file.

Reply:
In the mentioned PR I can see that the DataAccessError is handled for ES absence during unsupported features check. My PR proposes to identify and isolate all database framework errors and convert them to 503 (Service temporary unavailable) instead of 500, which gives the user no clue about what happened. 
I suggest to merge both of our fixes.